ada/views.py

```diff
@@ -47,8 +48,9 @@ def clickActivity(request: HttpRequest, pk: int, whatsappid: str) -> HttpRespons
     else:
         store_url_entry = RedirectUrlsEntry(symptom_check_url=redirect_url)
         store_url_entry.save()
+        customizationId = settings.ADA_CUSTOMIZATION_ID
```

Comment:
small one, but why not use _ instead of camelcase

Reply:
That's fine